project/src/components/card/card.tsx

```diff
@@ -31,14 +91,14 @@ export default function Card (): JSX.Element {
         </div>
         <div className="place-card__rating rating">
           <div className="place-card__stars rating__stars">
-            <span style={{width: '80%'}}></span>
+            <span style={{width: convertRatingToPercent(rating)}}></span>
             <span className="visually-hidden">Rating</span>
           </div>
         </div>
         <h2 className="place-card__name">
-          <a href="/">Beautiful &amp; luxurious apartment at great location</a>
+          <Link to={`offer/${id}`}>{title}</Link>
         </h2>
-        <p className="place-card__type">Apartment</p>
+        <p className={`place-card__type ${styles.placeCard__type}`}>{type}</p>
```

Comment:
```suggestion
        <p className={cn('place-card__type', styles.placeCard__type)}>{type}</p>
```
где cn дефолтный импорт библиотеки classnames